homeassistant/components/device_tracker/bluetooth_tracker.py

```diff
@@ -99,9 +100,17 @@ def update_bluetooth(now):
                 see_device(mac, result, rssi)
         except bluetooth.BluetoothError:
             _LOGGER.exception("Error looking up Bluetooth device")
-        track_point_in_utc_time(
-            hass, update_bluetooth, dt_util.utcnow() + interval)
+        if not once:
+            track_point_in_utc_time(
+                hass, update_bluetooth, dt_util.utcnow() + interval)
+
+    def handle_update_bluetooth(call):
+        """Update bluetooth devices on demand."""
+        now = dt_util.utcnow()
+        update_bluetooth(now, True)
 
     update_bluetooth(dt_util.utcnow())
 
+    hass.services.register(DOMAIN, "bluetooth_update", handle_update_bluetooth)
```

Comment:
Prefix the service name with the name of the platform, eg like this: `'bluetooth_tracker_update'`. I intentionally stripped the name of the second bluetooth.

Reply:
Makes sense to me, I updated it to ```'bluetooth_tracker_update'``` in a6d6f76